src/DssSpell.t.sol

```diff
@@ -132,45 +128,21 @@ contract DssSpellTest is GoerliDssSpellTestBase {
         //}
     }
 
-    function test_oracle_list() private {  // make private to disable
+    function testOracleList() private {  // make private to disable
         // address ORACLE_WALLET01 = 0x4D6fbF888c374D7964D56144dE0C0cFBd49750D3;
 
         //assertEq(OsmAbstract(0xF15993A5C5BE496b8e1c9657Fd2233b579Cd3Bc6).wards(ORACLE_WALLET01), 0);
 
-        //vote(address(spell));
-        //scheduleWaitAndCast(address(spell));
+        //_vote(address(spell));
+        //_scheduleWaitAndCast(address(spell));
         //assertTrue(spell.done());
 
         //assertEq(OsmAbstract(0xF15993A5C5BE496b8e1c9657Fd2233b579Cd3Bc6).wards(ORACLE_WALLET01), 1);
     }
 
-    function testSpellIsCast_GENERAL() public {
-        string memory description = new DssSpell().description();
-        assertTrue(bytes(description).length > 0, "TestError/spell-description-length");
-        // DS-Test can't handle strings directly, so cast to a bytes32.
-        assertEq(stringToBytes32(spell.description()),
-                stringToBytes32(description), "TestError/spell-description");
-
-        if(address(spell) != address(spellValues.deployed_spell)) {
-            assertEq(spell.expiration(), block.timestamp + spellValues.expiration_threshold, "TestError/spell-expiration");
-        } else {
-            assertEq(spell.expiration(), spellValues.deployed_spell_created + spellValues.expiration_threshold, "TestError/spell-expiration");
-        }
-
-        assertTrue(spell.officeHours() == spellValues.office_hours_enabled, "TestError/spell-office-hours");
-
-        vote(address(spell));
-        scheduleWaitAndCast(address(spell));
-        assertTrue(spell.done(), "TestError/spell-not-done");
-
-        checkSystemValues(afterSpell);
-
-        checkCollateralValues(afterSpell);
-    }
-
     function testRemoveChainlogValues() private { // make private to disable
```

Comment:
nit: I would move this test after the `testNewChainlogValues`, but maybe a reordering could be done later in a separate PR

Reply:
Yes let's please move on.